arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,870 @@
+    "application/json"
+  ],  
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
+    "/providers/microsoft.visualstudio/operations": {
+      "get": {
+        "tags": [
+          "accountResourceOperations"
+        ],
+        "summary": "Operations_List",
+        "description": "Gets the details of all operations possible on the Microsoft.VisualStudio resource provider.",
+        "operationId": "operations_List",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the possible operations on the Microsoft.VisualStudio resource provider.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "getOperationResources_List": {
+                "$ref": "../examples/GetOperations.json"
+            }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/microsoft.visualstudio/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "nameAvailability"
+        ],
+        "summary": "Accounts_CheckNameAvailability",
+        "description": "Checks if the specified Visual Studio Team Services account name is available. Resource name can be either an account name or an account name and PUID.",
+        "operationId": "accounts_CheckNameAvailability",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The request data.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NameAvailabilityPostRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the result of the name availability check.",
+            "schema": {
+              "$ref": "#/definitions/NameAvailabilityResponse"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "checkNameAvailability": {
+                "$ref": "../examples/CheckNameAvailability.json"
+            }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.visualstudio/account": {
+      "get": {
+        "tags": [
+          "accountResourceGroup"
```

Comment:
i have seen that usually the tags will be the name of the collection resource type (i.e. method group name in the operationId, in case of this operation `Accounts`). Please update tags associated with all operations in the swagger to match with method group name.